chapter_deep-learning-computation/use-gpu_vn.md

```diff
@@ -438,7 +445,11 @@ and log the matrix norm $\mathrm{tr} M M^\top$ one result at a time vs. keeping
 4. Measure how much time it takes to perform two matrix-matrix multiplications on two GPUs at the same time vs. in sequence on one GPU (hint: you should see almost linear scaling).
 -->
 
-*dịch đoạn phía trên*
+1. Thử một tác vụ có khối lượng tính toán lớn, ví dụ như nhân các ma trận kích thước lớn, để thấy sự khác nhau về tốc độ giữa CPU và GPU.
+Và với trường hợp tác vụ với khối lượng tính toán nhỏ thì sao?
+2. Nên làm thế nào để đọc và ghi các tham số của mô hình trên GPU?
+3. Đo thời gian thực hiện 1000 phép nhân ma trận kích thước $100 \times 100$ và trả về giá trị chuẩn $\mathrm{tr} M M^\top$ của từng kết quả, so sánh với việc lưu tất cả giá trị chuẩn tại một bản ghi ở GPU và chỉ trả về bản ghi đó.
```

Comment:
```suggestion
3. Đo thời gian thực hiện 1000 phép nhân ma trận kích thước $100 \times 100$ và ghi lại giá trị chuẩn $\mathrm{tr} M M^\top$ của từng kết quả, rồi so sánh với việc lưu tất cả giá trị chuẩn tại một bản ghi ở GPU và chỉ trả về bản ghi đó.
```

Reply:
Cảm ơn anh, em nghĩ thế này sẽ rõ nghĩa hơn, anh review xem có vấn đề gì không giúp em ạ.
```suggestion
3. Đo thời gian thực hiện 1000 phép nhân ma trận kích thước $100 \times 100$ và ghi giá trị chuẩn $\mathrm{tr} M M^\top$ của từng kết quả ra cửa sổ dòng lệnh, rồi so sánh với việc lưu tất cả giá trị chuẩn tại một bản ghi ở GPU và chỉ in ra cửa sổ dòng lệnh bản ghi đó.
```

Reply:
hmm đâu nhất thiết phải in log ra màn hình đâu, mình lưu trong file cũng được mà